react/features/base/participants/actions.js

```diff
@@ -9,6 +9,52 @@ import {
 import { getLocalParticipant } from './functions';
 
 /**
+ * Action to update a participant's avatar id.
+ *
+ * @param {string} id - Participant's id.
+ * @param {string} avatarId - Participant's avatar id.
+ * @returns {{
+ *      type: PARTICIPANT_UPDATED,
+ *      participant: {
+ *          id: string,
+ *          avatarId: string,
+ *      }
+ * }}
+ */
+export function changeParticipantAvatarID(id, avatarId) {
```

Comment:
If I understand it correctly the avatarId is generated once in Settings and then stays the same forever, so what exactly are we doing here?